docs/src/pages/[platform]/components/menu/examples/basicExample.tsx

```diff
@@ -1,18 +1,11 @@
-import { Divider, Menu, MenuItem } from '@aws-amplify/ui-react';
+import { Menu, MenuItem } from '@aws-amplify/ui-react';
 
 export const BasicExample = () => {
   return (
     <Menu>
-      <MenuItem onClick={() => alert('Download')}>Download</MenuItem>
```

Comment:
This does feel cleaner for the BasicExample, but I think we still need add a section that shows how to add interaction to the Menu options.